autoload/kite.vim

```diff
@@ -118,7 +118,12 @@ function s:setup_events()
     autocmd InsertCharPre            <buffer> call kite#completion#insertcharpre()
     autocmd TextChangedI             <buffer> call kite#completion#autocomplete()
 
-    autocmd CompleteDone             <buffer> call kite#snippet#complete_done()
+    if &ft == 'go'
+      autocmd CompleteDone           <buffer> call kite#completion#expand_newlines()
```

Comment:
We are planning on starting to return snippet completions with newlines in the near future. Will this work?

Reply:
It won't work with the current implementation, which tracks the start and end column numbers of the placeholders but assumes everything is on one line.  Perhaps we could open a separate issue for that?

Reply:
ok, yes I think we can open a separate issue, and have a discussion about the best way forward.

(perhaps we could reuse ultisnips or some other snippets solution if it's complicated to get working?)

Reply:
I looked at existing snippets plugins before building ours.  None offer the behaviour we want, e.g. navigating between snippets backward as well as forward.  Certainly none handle nested snippets.